src/components/Charts/Bars/Bars.js

```diff
@@ -0,0 +1,102 @@
+import React, { Component } from 'react'
+import _get from 'lodash.get'
+import { chartElementDefaultProps, chartElementPropTypes, getTooltipContent, scaleZero, getKey } from '../helpers'
+
+export default class Bars extends Component {
+  static propTypes = chartElementPropTypes
+  static defaultProps = chartElementDefaultProps
+
+  state = {
+    activeKey: this.props.activeChartElementKey,
+  }
+
+  shouldComponentUpdate = () => {
+    return true
+  }
```

Comment:
how does this work? lol

Reply:
This actually used to have more in it then got removed so I can take this out.